### PR TITLE
Slim down standard assets

### PR DESCRIPTION
**Pull request by davidkline-ms**
The standard assets package is intended to be common resources used by applications, including:

* UI acknowledgment sounds
* Common font(s)
* Standard color palette materials
* etc

Additional assets used in the example scenes should reside in the extended assets package.

This change moves example scene models to the appropriate package.

### CODE REVIEW

File: com.microsoft.mrtk.extendedassets/Models/Model_PushButton.fbx.meta [hunk 1]
```diff
@@ -1,26 +1,14 @@
 fileFormatVersion: 2
-guid: 2e4e683eccb3efe49ba775ada36cbc8f
```
Comment (keveleigh) [inline com.microsoft.mrtk.extendedassets/Models/Model_PushButton.fbx.meta:2]: Oh, what's the change here? Did we have Model_PushButton.fbx in both Standard and Extended? Hopefully they were the same 😬

Comment (keveleigh) [reply com.microsoft.mrtk.extendedassets/Models/Model_PushButton.fbx.meta:2]: Also, I _could_ see a button type being considered "standard" imo

Comment (davidkline-ms) [reply com.microsoft.mrtk.extendedassets/Models/Model_PushButton.fbx.meta:2]: we did have two copies. the models were identical and the guids different.

the interesting thing is that the round button was in both places and the square in extended assets

Comment (keveleigh) [reply com.microsoft.mrtk.extendedassets/Models/Model_PushButton.fbx.meta:2]: Oooh the square already being in extended assets is good enough signal for me 😜

**Review by Zee2 — APPROVED** [review]

**Pull request merged**